### PR TITLE
Doc web UI setting for device timeout calculation

### PR DESCRIPTION
**Pull request by kprosise**
New feature added as pager under UG:account and factory management. Updated section landing page and site index.

QA Steps: Ran pages through the linter and addressed issues. Rendered pages as html and ran linkcheck. No issues found.

This addresses Jira FFTK-2035.

Signed-off-by: Katrina Prosise <katrina.prosise@foundries.io>

## Readiness

* [x] Merge (pending reviews)
* [ ] Merge after _date or event_
* [ ] Draft

## Overview

New feature currently available in the web UI

## Checklist

* [x] Run spelling and grammar check, preferably with linter.
* [x] Avoid changing any header associated with a link/reference.
* [x] Step through instructions (or ask someone to do so).
* [x] Review for [wordiness](https://languagetool.org/insights/post/wordiness/)
* [x] Match tone and style of page/section.
* [x] Run `make linkcheck`.
* [x] View HTML in a browser to check rendering.
* [x] Use [semantic newlines](https://bobheadxi.dev/semantic-line-breaks/).
* [x] follow best practices for commits.
  * [x] Descriptive title written in the imperative.
  * [x] Include brief overview of QA steps taken.
  * [x] Mention any related issues numbers.
  * [x] End message with sign off/DCO line (`-s, --signoff`).
  * [x] Sign commit with your gpg key (`-S, --gpg-sign`).
  * [x] Squash commits if needed.
* [x] Request PR review by a technical writer and at least one peer.

## Comments

We could also put this under a `Settings` subsection or similar, but as it would be the only page at the moment, I put it at the same level as the other pages under `Factory & Account Management`

### CODE REVIEW

**Comment by doanac** [comment]
Docs for 16fbfef573cadc4207d8f8b1db955e1e5a4a47ad are browsable at: https://ci.foundries.io/projects/fio-docs/builds/1925/docs/artifacts/html/index.html

**Comment by kprosise** [comment]
linkcheck failure is unrelated to changes.
Appears to be a transient issue that has resolved itself.

**Review by MiloCasagrande — COMMENTED** [review]
Looks good overall to me.
Maybe add something about the "restore to default" button as well? (the arrow button on the right)

It will restore back to the default value of 4 hours (and you still need to save afterwards).

File: source/user-guide/account-management/setting-device-offline.rst [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+Accessible only through the web UI, Admin and Owner accounts may set the time until devices are considered to be offline.
+This can be set as an integer value for either hours or minutes.
+Setting the value to 0 disables the offline calculation.
+
+How It Works: Walk Through
+--------------------------
+
+From your Factory page, navigate to the :guilabel:`Settings` tab.
+Under :guilabel:`Prefences` is the :guilabel:`Offline device timeout` field.
```
Comment (vanmaegima) [inline source/user-guide/account-management/setting-device-offline.rst:14]:
```suggestion
Under :guilabel:`Preferences` is the :guilabel:`Offline device timeout` field.
```

File: source/user-guide/account-management/setting-device-offline.rst [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+.. figure:: /_static/userguide/account-management/device-offline.png
+   :align: center
+   :alt: example image showing the device offline settings tab
+
+Here you can enter the value, and select minutes or hours.
+Click :guilabel:`save`.
```
Comment (vanmaegima) [inline source/user-guide/account-management/setting-device-offline.rst:21]:
```suggestion
Click :guilabel:`Save`.
```

**Review by vanmaegima — APPROVED** [review]
Minor comments, but LGTM

**Comment by doanac** [comment]
Docs for 25535f2d36bfb2653286f5f8815327e2c422e270 are browsable at: https://ci.foundries.io/projects/fio-docs/builds/1928/docs/artifacts/html/index.html

File: source/user-guide/account-management/setting-device-offline.rst [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+From your Factory page, navigate to the :guilabel:`Settings` tab.
+Under :guilabel:`Preferences` is the :guilabel:`Offline device timeout` field.
+
+.. figure:: /_static/userguide/account-management/device-offline.png
+   :align: center
+   :alt: example image showing the device offline settings tab
+
+Here you can enter the value, and select minutes or hours.
+Click :guilabel:`Save`.
```
Comment (kprosise) [inline source/user-guide/account-management/setting-device-offline.rst:21]:
```suggestion
Click :guilabel:`Save`.

To restore the default value, use the button with the circle arrow, immediately to the right of the minute/hour selector.
After restoring to default, you will want to save your choice again.
```

**Comment by doanac** [comment]
Docs for 92186894e55fa36fe530053ed198a308ea21a4ae are browsable at: https://ci.foundries.io/projects/fio-docs/builds/1929/docs/artifacts/html/index.html

**Comment by doanac** [comment]
Docs for 345ad9023701692f8b5b3d7dc46f7b219b2f7438 are browsable at: https://ci.foundries.io/projects/fio-docs/builds/1930/docs/artifacts/html/index.html

**Pull request merged**